src/cell-info.vue

```diff
@@ -29,15 +29,15 @@
 
 			tr(v-if ="sensor.o.id != undefined")
 				td(style="text-align:left;")
-					a(:id="'graph_'+sensor.o.id+'_on'" class="graph_on" onclick="var sensor=this.id.substring(0,this.id.length-3);document.getElementById(sensor).style.display='';document.getElementById(sensor+'_on').style.display='none';document.getElementById(sensor+'_off').style.display=''; document.getElementById('images_'+sensor.substr(6)).innerHTML='<iframe src=\"https://grafana.aq.sensors.africa/d-solo/vYpoziaiz/sensors-africa-single-sensor-view?orgId=1&var-sensor_id=' + sensor.substring(6) + '&panelId=2\" width=\"290\" height=\"200\" frameborder=\"0\"></iframe><br /><iframe src=\"https://grafana.aq.sensors.africa/d-solo/vYpoziaiz/sensors-africa-single-sensor-view?orgId=1&var-sensor_id=' + sensor.substring(6) + '&panelId=3\" width=\"290\" height=\"200\" frameborder=\"0\"></iframe><br /><br />'; return false;" href='#' style='color:white; text-decoration: none;') (+)&nbsp;{{sensor.o.id}}
+					a(:id="'graph_'+sensor.o.id+'_on'" class="graph_on" onclick="var sensor=this.id.substring(0,this.id.length-3);document.getElementById(sensor).style.display='';document.getElementById(sensor+'_on').style.display='none';document.getElementById(sensor+'_off').style.display=''; document.getElementById('images_'+sensor.substr(6)).innerHTML='<iframe src=\"https://grafana.aq.sensors.africa/d-solo/vYpoziaiz/sensors-africa-single-sensor-view?orgId=1&var-sensor_id=' + sensor.substring(6) + '&panelId=2\" width=\"490\" height=\"200\" frameborder=\"0\"></iframe><br /><iframe src=\"https://grafana.aq.sensors.africa/d-solo/vYpoziaiz/sensors-africa-single-sensor-view?orgId=1&var-sensor_id=' + sensor.substring(6) + '&panelId=3\" width=\"490\" height=\"200\" frameborder=\"0\"></iframe><br /><br />'; return false;" href='#' style='color:white; text-decoration: none;') (+)&nbsp;{{sensor.o.id}}
 					a(:id="'graph_'+sensor.o.id+'_off'" class="graph_off" onclick="var sensor=this.id.substring(0,this.id.length-4);document.getElementById(sensor).style.display='none';document.getElementById(sensor+'_on').style.display='';document.getElementById(sensor+'_off').style.display='none'; document.getElementById('images_'+sensor.substr(6)).innerHTML=''; return false;" href='#' style='color:white; text-decoration: none; display: none;') (-)&nbsp; {{sensor.o.id}}
```

Comment:
Oh gosh ummm, could you move this onclick string function to be a code function s'il vous plait?... @nerdlet